api/backend/tags/routes.py

```diff
@@ -1,37 +1,48 @@
-from .. import api
+from .. import api, db
 from flask import Blueprint, request, session
 from flask_restful import Resource
 from ..models import Tag, User
-from ..tags.decorators import tag_exists
+from ..tags.decorators import tag_exists, validate_tag_form
 from ..tags.schemas import tag_schema, tags_schema, tag_form_schema
 from ..tags.utils import create_tag, update_tag
 
-
 # Blueprint for tags
 tag_bp = Blueprint("tags", __name__)
 
 
-# Class for updating, deleting & fetching tags
-class TagCRUD(Resource):
-    
-    # Fetch specific tag 
-    def get(self, tag_id):
-        tag = Tag.query.get(tag_id)
-
-        return tag_schema.dump(tag)    
+# Class to fetch and create tag
+class Tags(Resource):
+    @validate_tag_form
+    # Create Tag 
+    def post(self):
+        form_data = request.get_json()
+        tag = create_tag(form_data)
 
-    # Delete specific tag 
-    def delete(self, tag_id):
-        tag = Tag.query.get(tag_id)
-        db.session.delete(tag)
+        db.session.add(tag)
         db.session.commit()
 
-        return { 
-            "message": "Tag successfully deleted" 
-        }, 200
+        return {
+            "message": "Tag sucessfully created"
+        }, 201
 
 
+    # Fetch Tags
+    def get(self):
+        tag = Tag.query.limit(15).all()
+      
+        if len(tag) != 0:
+            return tags_schema.dump(tag)
+        else:
+            return {
+                "message": "No Tags"
+            }, 404
+
+
+# Class for updating, deleting & fetching tags
+class TagCRUD(Resource):
+
     # Update specific tag  
+    @tag_exists
```

Comment:
We should use the validate_tag_form decorator here as well since we have to make sure that the data that user's send in is sanitized